pkg/kp/kv.go

```diff
@@ -50,6 +53,42 @@ func (err KVError) Error() string {
 	return fmt.Sprintf("%s failed for path %s", err.Op, err.Key)
 }
 
+func (c consulStore) PutHealth(res map[string]string) (time.Time, time.Duration, error) {
+	key := fmt.Sprintf("%s%s/%s", res["service"], "Health", res["node"])
```

Comment:
Lets consolidate the path making function since this is repeated below and will be useful for others.

`func HealthPath(service, node string) string`